packages/grid/_modules_/grid/hooks/core/useGridStateInitialization.ts

```diff
@@ -105,11 +110,11 @@ export const useGridStateInitialization = (apiRef: GridApiRef, props: DataGridPr
 
   const forceUpdate = React.useCallback(() => rawForceUpdate(() => apiRef.current.state), [apiRef]);
 
-  const stateApi: Omit<GridStateApi, 'state'> = {
+  const stateApi: any = {
```

Comment:
TypeScript complains in line 119 when calling `useGridApiMethod`.
I did my best to avoid this `any` and have the Typing exactly work with the various plans whenever possible, but for this line I did not succeed :sob:

Reply:
Couldn't figure out why it's like that as well :/